examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
+sudo pkill -x openshift
+
+echo "Stopping all k8s docker containers on host ..."
+sudo docker ps --format='{{.Names}}' | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r sudo docker stop
```

Comment:
Nit, but I feel like `grep` is better here, the `awk` expression is pretty gnarly...

```sh
sudo docker ps --format='{{.Names}}' | grep -E '^k8s_' | xargs -l -r sudo docker stop
```